third_party/txt/src/skia/paragraph_skia.cc

```diff
@@ -79,6 +82,20 @@ class DisplayListParagraphPainter : public skt::ParagraphPainter {
     }
     size_t paint_id = std::get<PaintID>(paint);
     FML_DCHECK(paint_id < dl_paints_.size());
+
+    if (ShouldRenderAsPath(dl_paints_[paint_id])) {
```

Comment:
Consider putting this inside the `impeller_enabled_` block below, and FML_DCHECK'ing instead.

Reply:
I think you also want to guard this with `IMPELLER_SUPPORTS_RENDERING`?

Reply:
Done